app/Http/Controllers/Secretariat/UserController.php

```
@@ -245,20 +260,15 @@ public function updateEducationalInformation(Request $request, User $user): Redi
      */
     public function updateAlfonsoStatus(Request $request, User $user)
     {
-        $this->authorize('view', $user);
+        $this->authorize('editStaticEducationalInformation', $user);
         session()->put('section', 'alfonso');
 
-        $validator = Validator::make($request->all(), [
+        $data = $request->validate([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (Unused Code Rules)

(UnusedLocalVariable)